project/Version.scala

```diff
@@ -2,7 +2,7 @@ object Version {
   val awscala = "0.8.1"
   val vectorpipe = "1.2.0-SNAPSHOT"
```

Comment:
I want to suggest a version number bump here, either to a new major version number or to apply a "-GT3" suffix.  I'm not sure I know what semver would have to say in this case.

Reply:
Looks like an externally visible breaking change would be the swap from GT geom types to JTS? Along with some import changes, such as `store` and `layer`? 

If we're semvering and there's breaking changes, that's a major version bump. If we need to do bugfix releases supporting gt 2 those can continue to be minor version or bugfix changes on VP 1.0.

So I'd definitely agree with a version bump, and additionally suggest major bump to VP 2.0.0